### PR TITLE
MWPW-159028: Consume aem-fragment from Milo

### PR DESCRIPTION
**Pull request by yesil**
Milo PR: https://github.com/adobecom/milo/pull/2952

Test URLs:
- Before: https://main--mas--adobecom.hlx.page
- After: https://mwpw-159028--mas--adobecom.hlx.page?milolibs=mwpw-159028--milo--adobecom

### CODE REVIEW

**Review by npeltier — APPROVED** [review]
lgtm (once merged with variant picker ;) )

**Review by Axelcureno — APPROVED** [review]
lgtm

**Pull request merged**
